doc/userguide/partials/options.rst

```diff
@@ -300,3 +300,12 @@
 .. option:: --simulate-ips
 
    Simulate IPS mode when running in a non-IPS mode.
+
+.. option:: --enable-etm
```

Comment:
I think I would prefer limiting configurability to the yaml. @jasonish thoughts?

Reply:
I agree.  Config file option.  It can then be `--set` if needed.